### ISSUE TITLE
Crash when notarizing app bundle without path prefix

### ISSUE THREAD

**Issue by PatTheMav** [akeru-inc/xcnotary#6]
With the current version notarizing `.dmg` images works flawlessly, however trying to notarize an app bundle fails with the following message:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/notarize/run.rs:116:22
``` 

The pre-checks all finish successfully, but it seems when the zip is created for upload the `ditto` step breaks.

EDIT: It seems that running `xcnotary` pointing to a file in the same directory without `./` as a prefix leads to this issue.

`xcnotary notarize MY_APP.app` fails, `xcnotary notarize ./MY_APP.app` works.

**Comment by davidvartan** [comment]
I can repro this. Thanks for the help tracking this down!  rust-lang/rust/issues/36861

**Comment by davidvartan** [comment]
Fixed in v0.4.1
